MANIFEST.in

```diff
@@ -1,7 +1,8 @@
-exclude *.yml
-exclude *.md
-exclude *.txt
-exclude .flake8
-exclude .coveragerc
-exclude .gitignore
-include edk2toolext/bin/NuGet.exe
\ No newline at end of file
+exclude *.yml
+exclude *.md
+exclude *.txt
+exclude .flake8
+exclude .coveragerc
+exclude .gitignore
+include edk2toolext/bin/NuGet.exe
+include *.txt *.md
```

Comment:
it seems odd that we "exclude" these two types on line 2 and 3 and then include them here.  

I think we need a more specific pattern.  Also looking at what is included i don't think line 2/3 are doing what we expected.

Reply:
Feel free to take over, it has "allow edits by maintainers" checked.  I don't know what's best for your project, only that this makes it build again...

Reply:
Thanks. I appreciate the effort in finding the issue, reporting the issue, and submitting the PR.